src/models.js

```diff
@@ -25,7 +24,24 @@ exports.getModels = function(options, callback) {
   sttRequest.setRequestHeader('X-Watson-Authorization-Token', token);
   sttRequest.onload = function() {
     var response = JSON.parse(sttRequest.responseText);
-    callback(response.models);
+    var sorted = response.models.sort(function(a,b) {
+    if(a.name > b.name) {
+      return 1;
+    }
+    if( a.name < b.name) {
+      return -1;
+    }
+    return 0;
+    });
+    response.models=sorted;
+    localStorage.setItem('models', JSON.stringify(response.models));
+    var viewContext1 = {
+      currentModel: 'en-US_BroadbandModel',
```

Comment:
German, that is our main model, it is the default model for English, that one stays. (we only need to remove Japanese models and Michael's voice for TTS)